### PR TITLE
Filter

### PR DESCRIPTION
**Pull request by kotalwar-krishna**
Filter related Css you find in filter.css and releated HTML you find in product.html files nav bar.

**Screenshots**
img11

### CODE REVIEW

File: Screens/filter.css [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+}
+
+.filter-form .filter-category{
+    font-size: 1.4rem;
+    text-align: start;
+    padding-left: 1rem ;
+}
+
+.clear-filter {
+    color: black;
```
Comment (psychoactiv) [inline Screens/filter.css]: Color variables would be preferred here.

**Review by psychoactiv — APPROVED** [review]
Code is good to merge just do the changes suggested

File: home-css/modifiction_nav_CL.css [hunk 2]
```diff
@@ -94,4 +94,9 @@
   .comp-header .login-form p a{
     color:var(--orange);
     text-decoration: underline;
-  }
\ No newline at end of file
+  }
+
+  /* .comp-header .filter-form.active{
```
Comment (Kothalkarpriya) [inline home-css/modifiction_nav_CL.css]: remove commented code if it is not needed

**Review by Kothalkarpriya — APPROVED** [review]
Good Work You can merge

File: Screens/filter.css (hunk 1 above)
Comment (kotalwar-krishna) [reply Screens/filter.css]: yes i will do

File: home-css/modifiction_nav_CL.css (hunk 2 above)
Comment (kotalwar-krishna) [reply home-css/modifiction_nav_CL.css]: not needed i will remove it

**Pull request merged**